content/en/02-installing-pixie/04-install-schemes/01-cli.md

```diff
@@ -103,3 +103,7 @@ Certain labels are reserved for internal use by Pixie. The following are reserve
 - `“component"`
 - `“vizier-updater-dep"`
 - `“app"`
+
+### More Deploy Options
```

Comment:
What about moving the section above ("Labeling resources") to the new Admin > Deploy Options section?